2_Operations/user-stories.md

```diff
@@ -36,37 +37,36 @@ Finally, take a bit of time to consider at least one pitfall (downside) to imple
 
 ## Story Points
 
-A [Story Point](https://agilefaq.wordpress.com/2007/11/13/what-is-a-story-point/) is an arbitrary measure used by Scrum teams to indicate the effort required to implement a User Story. One full-time member on sprint should be able to do 10 Story Points. That said, there should be 2 Story Points reserved for code reviews, cleanup, writing new User Stories, catchups, mentoring, and pair programming.
+A [Story Point](https://agilefaq.wordpress.com/2007/11/13/what-is-a-story-point/) is an arbitrary measure used by Scrum teams to indicate the effort required to implement a User Story. Story points are not a representation of time, but rather a combination of Risk, Repetition, and Complexity. However, we typically think of one story point as the amount of work a person can do in one day.
 
 ## Pipelines
 
-From initial creation to being worked upon and finally closed, a User Story will be moved to different pipelines as it progresses through a sprint. This is a summary of the workflow of a User Story:
+From initial creation to being worked upon and finally closed, a User Story will be moved to different pipelines as it progresses through the Kanban board. This is a summary of the workflow of a User Story:
 
 1. **New Issues** - New work-in-progress (WIP) User Stories, also includes bugs or issues.
 1. **Estimation** - Defined User Stories, ready for verifying and Story point estimation.
-1. **Backlog** - Waiting for inclusion in a sprint. They are adequately defined with assigned Story points.
-1. **To Do** - In the current sprint with a champion assigned to work on it.
-1. **In Progress** - Being worked upon by the champion during the current sprint.
+1. **Backlog** - Waiting to be started. They are adequately defined with assigned Story points, and is typically also assigned to a Nitean.
+1. **In Progress** - Currently being worked upon by the assignee.
 1. **Review** - Completed with a demo uploaded and awaiting final review.
 1. **Closed** - All defined expectations have been met.
 
 ### Estimation
 
-For the User Story to be included in a sprint, it must be verified and voted upon. To start this process put the User Story in the `Estimation` pipeline.
+For the User Story to be moved to the **Backlog**, it must be verified and voted upon. To start this process put the User Story in the **Estimation** pipeline.
 
-The Scrum master and product owner will verify that the User Story is well defined. If required they will assign people from whom they want to receive feedback for better definition of the User Story. Once these assignees post their feedback, they remove their assignment from the User Story.
+The project lead and product owner will verify that the User Story is well defined. If required they will assign people from whom they want to receive feedback for better definition of the User Story. Once these assignees post their feedback, they remove their assignment from the User Story.
```

Comment:
```suggestion
The project lead will verify that the User Story is well defined. If required they will assign people from whom they want to receive feedback for better definition of the User Story. Once these assignees post their feedback, they remove their assignment from the User Story.
```

I don't think we need to mention product owner anymore. Project lead is fine for all these.